profile/README.md

```diff
@@ -1,17 +1,28 @@
 # Project-Origin
+<!--
+The mission of the project in as a concise format as possible. It should answer what the project's purpose is and enable contributors to make a good first guess whether a suggested feature will likely be in scope for the project, or not.
+-->
 
-Project-Origin aims to create solutions to enable publicly verifiable proofs for tracking of Energy.
+Project-Origin is an open source collaboration that aims to create a public permissioned distributed ledger where everyone can validate the origin of their electricity. By being able to do so, we can support and accelerate the green transition. The long-term goal is to expand the scope from electricity, to include other utilities, because we find that the functionality can create value in a broader scope. By creating an open source platform for collaboration, we can pool ressources and accelerate the development of critical digital infrastructure that is secure and the data verifiable. 
 
-## Registry
+The open source platform is set up with decision- and work structures, to achieve effective asynchronous collaboration through transparent, accessible software, and by using decision-making processes driven by \emph{doing}.
 
-The goal of [this project](https://github.com/orgs/project-origin/projects/1) is to create the first version of a **Federated Publicly Verifiable Registry**
+## Ongoing Implementation Efforts 
+There is currently an implementation effort working on creating the first version of a federated, publicly verifiable registry. Go to the [Registry repository](https://github.com/project-origin/registry) to learn more about the implementation details and project status. 
 
-The registry will enable the following:
+## Getting Involved
+<!--
+- A "Getting involved" section that explains which public, archived, linkable communication channels the project uses. This should include a link to the project issue tracker, but also to any further discussion media used.
+-->
 
-- [ ] Federation setup enabling multiple registries to create a common truth.
-- [ ] Enable the issuance of Granular Certificates.
-- [ ] Enable the claim of Granular Certificates to some consumption.
-- [ ] The registry handles ownership based on public-private key-pairs.
-- [ ] Users can verify that they have a proffable amount of a Granular Certificate, that mathematically proves it is only used once.
-- [ ] Users can transfer ownership of a sub-part of a certificate to another private-key.
-- [ ] Privacy of the individual measurement is hidden behind a Petersen-commitments.
+If you want to get involved in the project with us, the first important step is to make yourself familiar with our decision-making structures. To do that, we have made a guide on [How to Start Collaborating with Us](www.deadlink) for you. Further, you can take a look at our [Discussion Channels](https://github.com/orgs/project-origin/discussions), to obtain an overview of the project's current status and progress. 
```

Comment:
I would also say: Not only the decision making structures but also what already exists, is planned and the general lay of the land to evaluate if this is something I'd like to get into in more detail. See comment above for ideas.

```suggestion
If you want to get involved in the project with us, the first important step is to make yourself familiar with our decision-making structures. 
To do that, we have made a guide on [How to Start Collaborating with Us](www.deadlink) for you. 
Further, you can take a look at our [Discussion Channels](https://github.com/orgs/project-origin/discussions), to obtain an overview of the project's current status and progress. 
```
(I added a bunch of newlines. Markdown will still render this as one paragraph, but Git works on a line-by-line basis. So you can review/commit/change each line/sentence separately. (Mr. Lamport winks in the far distance :wink: )